x-pack/platform/plugins/shared/alerting/server/application/maintenance_window/methods/update/update_maintenance_window.ts

```diff
@@ -128,14 +128,6 @@ async function updateWithOCC(
         updatedAt: modificationMetadata.updatedAt,
       });
 
-    if (updateMaintenanceWindowAttributes.scopedQuery) {
```

Comment:
Why you are deleting this part?

Reply:
The long-term idea is to move away from this logic. Users want to specify queries but don't want to worry about solution/category.

Iirc in your PR there was also some discussion about this where I mentioned removing the validation here.